decidim-meetings/lib/decidim/meetings/test/factories.rb

```diff
@@ -33,10 +40,10 @@
     transparent { true }
     questionnaire { build(:questionnaire) }
     registration_form_enabled { true }
-    registration_terms { Decidim::Faker::Localized.wrapped("<p>", "</p>") { generate_localized_title } }
+    registration_terms { generate_localized_description(:meeting_registration_terms, skip_injection: skip_injection) }
     registration_type { :on_this_platform }
     type_of_meeting { :in_person }
-    component { build(:component, manifest_name: "meetings") }
+    component { build(:meeting_component, skip_injection: skip_injection) }
```

Comment:
This is how it's done in `develop` and in the original PR: 
```suggestion
    component { build(:meeting_component) }
```